src/DGtal/geometry/curves/AlphaThickSegmentComputer.h

```diff
@@ -0,0 +1,711 @@
+   * @param[out] nu the width of the strip. 
+   *
+   **/
+  void computeParalellStripParams(double &mu, PointD &N, double &nu) const;
+  
+  
+  /**
+   * @return the two values of the normal vector given as a Point(a,b)
+   * (the normal is oriented toward the segment by considering the
+   * edge PQ of the convexhull antipodal pair)..
+   **/
+  
+  PointD getNormal() const;
+
+  /**
+   * @return the mu parameter of the current segment.
+   **/
+  double getMu() const;
+ 
+ 
+  /**
+   * @return the nu parameter the of the current segment.
+   **/
+  double getNu() const;
+   
+  
+   /**
+   * @return the segment length defined from the bouding box (@see getBoundingBox).
+   **/
+  double getSegmentLength() const;
+  
+
+  /**
+   * @return 'true' if the points of the segment computer are stored in the main container.
+   **/
+  bool isStoringSegmentPoints() const;
+
+
+  /**
+   * @return the total number of points beeing recognized through in the segment construction.
+   **/  
+  unsigned int getNumberSegmentPoints() const;
+
+
+
+  // ------------------------- Display services ------------------------------
+
+
+  /**
+   * @return the current alpha thick segment convexhull given as a vector containing the vertex points.
+   **/
+  std::vector<InputPoint> getConvexHull() const;
+
+
+  
+  /**
+   * Computes the segment bounding box according to two extremity points (\a aFirstPt, \a aLastPt).
+   * @param[in] aFirstPt the first extrem point.
+   * @param[in] aLastPt the last extrem point. 
+   * @param[out] pt1LongestSegment1 the first point of one of the longest segment.
+   * @param[out] pt2LongestSegment1 the second point of one of the longest segment.
+   * @param[out] pt3LongestSegment2 the first point of one of the second longest segment.
+   * @param[out] pt4LongestSegment2 the second point of one of the second longest segment.
+   * @param[in] minVisibleWidthBounds the minimal width of the resulting bounding box (for drawing issue).
+   *
+   * @note the segment bounding box can be drawn with the sequence of
+   * out parameters pt1LongestSegment1, pt2LongestSegment1,
+   * pt3LongestSegment1, pt4LongestSegment1.
+   **/  
+  void getBoundingBoxFromExtremPoints(const InputPoint &aFirstPt,
+                                      const InputPoint &aLastPt,
+                                      PointD &pt1LongestSegment1,
+                                      PointD &pt2LongestSegment1,
+                                      PointD &pt3LongestSegment2,
+                                      PointD &pt4LongestSegment2,
+                                      double minVisibleWidthBounds = 0.2) const;
+  
+  
+  /**
+   * Computes the segment bounding box defined from the extremity
+   * points computed after a scan of the current convexhull. Note that
+   * this bouding box differs from the begin/end points bounding box
+   * when a large amount of noise are give in the initial curve.
```

Comment:
Taking only the end points of sequence belonging to the segment can be wrong in case of noise. 
The bounding box if defined according the main direction of the segment with the projection of contour point and upper/lower leaning points of the convexhull.
I edit the doc to precise that, but let me know if you think to a better method name.